test/solidity/errors/errors.js

```diff
@@ -53,4 +53,40 @@ describe('Solidity Errors', function () {
     }
   })
 
+  it('should confirm revert with custom error works', async function () {
+    try {
+      await contract.revertWithCustomError()
+    } catch (err) {
+        expect(err.code).to.equal('CALL_EXCEPTION')
+        expect(err.errorName).to.equal('InsufficientBalance')
+        expect(err.errorArgs.available).to.equal(ethers.BigNumber.from(1))
+        expect(err.errorArgs.required).to.equal(ethers.BigNumber.from(100))
+    }
```

Comment:
Please check that a revert occurs.
As written if no error is thrown the overall test passes.